src/quantum/setup.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from codecs import open
+from setuptools import setup, find_packages
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+# TODO: Confirm this is the right version number you want and it matches your
+# HISTORY.rst entry.
+VERSION = '0.11.2906.2'
```

Comment:
Same version problem mentioned above.

Reply:
Agreed, however this is going to be released and documented as part of the Quantum Development Kit (QDK), and has specific dependencies on the QDK version so it would greatly help us to keep the versions in-sync. As such, I'd rather keep it like this.